haproxy: configure max instances based on replicas (#1468)
element-hq/ess-helm: newsfragments/1468.fixed.md

> Fix an issue where HAProxy could not become aware of instances if replicas where superior to 20.

Reviewer: ```suggestion
Fix an issue where HAProxy could not route to more than 20 instances of a worker
```
